README.md

```diff
@@ -105,11 +105,10 @@ pp(result.records)
 
 ## Limitations
 Early days. Still a fair few rough edges.
-* No test coverage.
 * Does not support _without[field]_ searches... yet.
 * Oh, you know... all that other stuff.
 
 ##License
 
 GPLV2. 
-See [license.txt](license.txt)
\ No newline at end of file
+See [license.txt](license.txt)
```

Comment:
Used GitHub web UI to create this PR... no idea why it's highlighting this last line though.